test/dotnet-sln-add.Tests/GivenDotnetSlnAdd.cs

```diff
@@ -841,28 +889,28 @@ public void WhenPassedMultipleProjectsAndOneOfthemDoesNotExistItCancelsWholeOper
 
         //ISSUE: https://github.com/dotnet/sdk/issues/522
         //[Fact]
-        public void WhenPassedAnUnknownProjectTypeItFails()
```

Comment:
Rather than commenting this test method out, let's uncomment the `Fact` attribute so that it skips instead:

```c#
[Fact(Skip = "https://github.com/dotnet/sdk/issues/522")]
public void WhenPassedAnUnknownProjectTypeItFails()
{
...
```

Reply:
Hi @peterhuene. Thanks for the review! I've updated the attribute accordingly in c3527b3.